csi/deployment.go

```diff
@@ -200,6 +200,7 @@ func NewResizerDeployment(namespace, serviceAccount, resizerImage, rootDir strin
 		[]string{
 			"--v=5",
 			"--csi-address=$(ADDRESS)",
+			"--timeout=2m5s",
```

Comment:
Why we specify a timeout here?

Reply:
Since the default timeout is 10s and we now do verify that the volume expansion is completed, see comment answer to phan's question above, this is important to keep kubernetes state and longhorns state consistent.

The timeout we use for the waitForVolumeState is hardcoded at 2 minutes currently, so I add 5s extra :)
So that we can timeout the active call. This ensures that the external-resizer does not flood the csi driver with a bunch of unnecessary concurrent ExpandCalls for the same volume. Since we will return success as soon as the expansion is completed or timeout the call.

```go
	if !cs.waitForVolumeState(req.VolumeId, "volume expansion", volumeExpansionComplete, false, false) {
		return nil, status.Errorf(codes.DeadlineExceeded, "Expanding volume %s existing capacity %v requested capacity %v failed",
			req.GetVolumeId(), existingSize, requestedSize)
	}
```

Reply:
Can we add this explaination as a comment here (with a brief version)?

Reply:
sure thing, but we didn't add any comments when we changed the timeout value for the attacher :)